### ISSUE TITLE
"The app took too long to respond" on search

### ISSUE THREAD

**Issue by Wijkie76** [athombv/com.spotify#2]
The playlist are imported and playing just fine, but i can't search for a "single track" 
![spotify-beta](https://user-images.githubusercontent.com/28150252/29704344-b6740c32-8979-11e7-8634-6927c907994a.PNG)

I'm using this app with Spotify desktop. I don't own a Sonos. Running v1.5.0-rc8.

**Comment by casperboone** [comment]
Same issue here:

![image](https://user-images.githubusercontent.com/15815208/30003809-f563b9c8-90c3-11e7-85f4-3f03e51fa175.png)

Shortly after, the app crashed, maybe related:
```
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
```

**Comment by BasKiers** [comment]
Hi, 

This error should be fixed as of v1.1.3 of the Spotify app.

Regards,
Bas
